server/index.js

```diff
@@ -1,9 +1,57 @@
 const express = require('express')
 const app = express()
 const port = 3000
+const Review = require('../database/connection.js')
 
+app.use(express.json());
 app.use(express.static(__dirname + "/../public"));
 
+app.get("/api/reviews/1", (req, res) => {
+  var objWithColumnArrays = {
+    cleanliness: [],
+    accuracy: [],
+    communication: [],
+    checkin: [],
+    location: [],
+    value: []
+  };
+
+  return new Promise((resolve, reject) => {
+    Review.find({}, { cleanliness: 1, _id: 0 })
```

Comment:
Sorry if I just don't understand mongoose as well, but do all these queries need to be separate or can they be bunched into one?

Reply:
Other than that I think it looks great.  If youre getting a response from the server I say its approved!